File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/settings-new/billing/shared/dto/bill-service-item-scheme-setting.dto.ts
export class BillServiceItemSchemeSetting_DTO {
  public ServiceItemSchemeSettingId: number = 0;
  public SchemeId: number = 0;
  public ServiceItemId: number = 0;
  public ServiceItemCode: string = '';
  public ServiceItemName: string = '';
  public RegDiscountPercent: number = 0;
  public OpBillDiscountPercent: number = 0;
  public IpBillDiscountPercent: number = 0;
  public AdmissionDiscountPercent: number = 0;
  public IsCoPayment: boolean = false;
  public CoPaymentCashPercent: number = 0;
  public CoPaymentCreditPercent: number = 0;
  public CreatedBy: number = 0;
  public CreatedOn: string = null;
  public ModifiedBy: number = null;
  public ModifiedOn: string = null;
  public IsActive: boolean = true;
  public ServiceDepartmentId: number = 0;
  public itemIsSelected: boolean = false;

  public IsValidRegDiscountPercent: boolean = true;
  public IsValidOpBillDiscountPercent: boolean = true;
  public IsValidIpBillDiscountPercent: boolean = true;
  public IsValidAdmissionDiscountPercent: boolean = true;
  public IsValidCopayCashPercent: boolean = true;

  public IsValidCopayCreditPercent: boolean = true;
  public initialSelectionState: boolean;

}

File: Code/Websites/DanpheEMR/Controllers/Billing/Shared/BillServiceItemSchemeSetting_DTO.cs
using System;

namespace DanpheEMR.Controllers.Billing.Shared
{
    public class BillServiceItemSchemeSetting_DTO
    {
        public int ServiceItemSchemeSettingId { get; set; }
        public int SchemeId { get; set; }
        public int ServiceItemId { get; set; }

        public string ServiceItemCode { get; set; }

        public string ServiceItemName { set; get; }

        public int ServiceDepartmentId { set; get; }
        public decimal RegDiscountPercent { get; set; }
        public decimal OpBillDiscountPercent { get; set; }
        public decimal IpBillDiscountPercent { get; set; }
        public decimal AdmissionDiscountPercent { get; set; }
        public bool IsCoPayment { get; set; }
        public decimal CoPaymentCashPercent { get; set; }
        public decimal CoPaymentCreditPercent { get; set; }
        public int CreatedBy { get; set; }             
        public bool itemIsSelected { get; set; }
        public DateTime? CreatedOn { get; set; }
        public int? ModifiedBy { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public bool IsActive { get; set; }
    }
}
